osu.Game/Graphics/OsuIcon.cs

```diff
@@ -89,6 +89,8 @@ public static class OsuIcon
         public static IconUsage ModSpunOut => Get(0xe046);
         public static IconUsage ModSuddenDeath => Get(0xe047);
         public static IconUsage ModTarget => Get(0xe048);
-        public static IconUsage ModBg => Get(0xe04a);
+
+        // Use "Icons/BeatmapDetails/mod-icon" instead
+        // public static IconUsage ModBg => Get(0xe04a);
```

Comment:
I dunno. I left it here in case someone comes across the icon in the font and wonders why they can't use it.

I think this whole font will eventually get obsoleted..

Reply:
Hmm okay, sure.